tests/account_test.js

```diff
@@ -11,7 +11,9 @@ Scenario("I can verify that I am on the correct obit", ({ I, obitPage }) => {
   );
   // Then I should see her name in the heading
   I.see("Virginia", obitPage.nameHeader);
-});
+})
+  .tag("@JanusBeta")
+  .tag("@prodSafe");
 
```

Comment:
Not an issue for this project, but our repo uses @productionSafe, just FYI (and our tools enforce it if the environment we're pointing to is production)